src/drivers/uavcan/uavcan_servers.hpp

```diff
@@ -76,7 +76,7 @@
  */
 class __EXPORT UavcanServers
 {
-	static constexpr unsigned NumIfaces = 1;  // UAVCAN_STM32_NUM_IFACES
+	static constexpr unsigned NumIfaces = 2;  // UAVCAN_STM32_NUM_IFACES
```

Comment:
Does respecting `UAVCAN_STM32_NUM_IFACES` work here?

Reply:
Looks like it does.